.circleci/config.yml

```diff
@@ -307,6 +307,12 @@ jobs:
                   [ -f ~/project/.env ] && source ~/project/.env
                   [ -f ~/project/.env.ef_circleci ] && source ~/project/.env.ef_circleci
 
+                  # Copy .env.ef_circleci to .env in the output directory
+                  log "Copying environment variables to output directory..."
+                  if [ -f ~/project/.env.ef_circleci ]; then
+                    cp ~/project/.env.ef_circleci ~/project/repository/.env
```

Comment:
this would overwrite a .env file if it exists?

Reply:
yep